src/MessageHandler/ActivityPub/Inbox/CreateHandler.php

```diff
@@ -87,13 +88,16 @@ public function doWork(MessageInterface $message): void
                 $username = $e->actor->name;
             }
             $this->logger->info('Did not create the post, because the magazine {m} restricts posting to mods and {u} is not a mod', ['m' => $e->magazine, 'u' => $username]);
+        } catch (InstanceBannedException $e) {
+            $this->logger->error('Did not create the post, because the user\'s instance is banned');
```

Comment:
why not a `info`? It's not really an error.. Its more a notification.

Reply:
true